internal/controllers/topology/cluster/patches/variables/variables_test.go

```diff
@@ -66,12 +66,22 @@ func TestGlobal(t *testing.T) {
 						Class:   "clusterClass1",
 						Version: "v1.21.1",
 					},
+					ClusterNetwork: &clusterv1.ClusterNetwork{
+						APIServerPort: pointer.Int32(8),
+						Services: &clusterv1.NetworkRanges{
+							CIDRBlocks: []string{"10.10.10.1/24"},
+						},
+						Pods: &clusterv1.NetworkRanges{
+							CIDRBlocks: []string{"11.10.10.1/24"},
+						},
+						ServiceDomain: "cluster.local",
+					},
 				},
 			},
 			want: VariableMap{
 				"location":   toJSON("\"us-central\""),
 				"cpu":        toJSON("8"),
```

Comment:
Just fyi. I will migrate some of the other unit tests here to use `toJSONCompact` in a PR I'll open today. Just in case you want to migrate the Cluster tests too. It's so much easier to read / check if it's correct.

Reply:
If we aim to backport, let's move all the test refactoring to a follow up PR

Reply:
I've moved the cluster test to toJSONCompact - the only issue will be to rebase one of the variable-adding PRs on top of the other.

Reply:
Yup. I think we should be fine when we cherry-pick them in the same order.